### PR TITLE
added breed and picture

### LINKED ISSUE #84: Add a /breeds and /pictures endpoint to animals route.
**Describe the feature**
Add an endpoint  `GET` breed of a particular animal in /animals route.
Add an endpoint  `GET` pictures of a particular animal in /animals route.

**Expected behaviour**
Return array of breeds of a particular animal.
Return array of pictures of a particular animal.

**Are you working on this feature? (Yes)**

### PR DESCRIPTION
**Pull request by ritik307**
# Related Issue

-   closes #84 

# Proposed Changes

-   added `/animal/type?option=breed`
-   added `/animal/type?option=picture`

# Checklist

-   [x] Tests
-   [x] Translations

### CODE REVIEW

**Comment by ritik307** [comment]
@ShauryaAg @paramsiddharth  Plz review

File: src/api/routes/animals.ts [hunk 1]
```diff
@@ -21,12 +21,23 @@ router.get("/", (req: Request, res: Response, next: NextFunction) => {
...
+                    result = data.data[i].breed;
+                    break;
+                }
+                else if(option === "picture"){
+                    result = data.data[i].picture;
+                    break;
+                }
+                else{
+                    result = data.data[i];
+                    break;
```
Comment (ShauryaAg) [inline src/api/routes/animals.ts:39]: Please return `400` if `query.option` does not match `breed` or `picture`

File: src/api/routes/vehicles.ts [hunk 2]
```diff
@@ -14,4 +14,5 @@ router.get("/", (req: Request, res: Response, next: NextFunction) => {
     }
 });
 
+
```
Comment (ShauryaAg) [inline src/api/routes/vehicles.ts:17]: Please remove this unnecessary new line.

**Review by ShauryaAg — CHANGES_REQUESTED** [review]

**Comment by ritik307** [comment]
@ShauryaAg Done! Plz check

**Review by paramsiddharth — APPROVED** [review]

**Review by ShauryaAg — APPROVED** [review]

**Comment by paramsiddharth** [comment]
Thank you!

**Pull request merged**